module3/Projects/Mod3Project.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: page
+title: Mod3 Project
+---
+
+# Message Logger - The Final Iteration!
+
+## Learning Goals
+- Demonstrate:
+    * Version Control, including branches and pull requests.
+    * MVC code organization within .NET
+    * Feature testing of a .NET application
+
+- Explore:
+    * Usability and creating a quality user experience
+    * Planning out the implementation of a feature
+    * Using TDD to add additional features! (For any new feature that you create, try writing the tests first, and then creating the code that makes those tests pass!)
+
+## Project Description
+For this project, you will build the final iteration of the Message Logger we created in Mod1 and built on in Mod2. In Mod2 you added a database, and now you're going to add a frontend! You are encouraged to bring in code you wrote in the Mod2 project, especially for calculating message statistics.
+
+
+This project will give you the opportunity to showcase all of the concepts we have learned this Mod.  There might be skills that we covered all the way back in week 1, so don't forget to use those earlier lessons and labs as a resource!
+
+**Project Setup**
+To get started on this project:
+
+1. **fork and clone** this [starter repo](https://github.com/turingschool-examples/MvcMessageLogger)

```

Comment:
I started up this repo with no problem 🎉 I also like that you called out places to put their deliverables :)